config/environment.js

```diff
@@ -6,7 +6,7 @@ var VersionChecker = require('ember-cli-version-checker');
 module.exports = function( environment, appConfig, addon ) {
   appConfig['ember-collapsible-panel'] = appConfig['ember-collapsible-panel'] || {};
 
-  var checker = new VersionChecker(addon);
+  var checker = new VersionChecker(addon.project);
```

Comment:
Why is this looking at the project now? I would think that `addon.parent` is correct (because we don't require `liquid-fire` to be provided by only the app, we only care that the components will be resolvable)...

Reply:
On second thought, I think we can kill all of this custom config...

Reply:
I agree we can kill it. I was just trying to keep things simple but I will remove this.